client-runtime/src/main/java/com/microsoft/rest/v2/util/ByteBufferUtil.java

```diff
@@ -0,0 +1,28 @@
+/**
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License. See License.txt in the project root for
+ * license information.
+ */
+
+package com.microsoft.rest.v2.util;
+
+import java.nio.ByteBuffer;
+
+/**
+ * Contains helper methods for dealing with Flowables.
+ */
+public class ByteBufferUtil {
+    /**
+     * Convert the provided ByteBuffer to a byte[].
+     * @param byteBuffer The ByteBuffer to convert.
+     * @return The converted byte[].
+     */
+    public static byte[] toByteArray(ByteBuffer byteBuffer) {
```

Comment:
Do you think it would be useful to somehow convey that this is a copying operation and not a reference to the underlying `.array()`?

Reply:
I updated the comment to say that it is a read operation.